Basic Rules.txt

```diff
@@ -917,8 +1052,9 @@ in the 90-100 range caused by a raw roll or by adding your weapon use skill),
 the reload time is halved (for Bolt-Actions it is reduced to 1 Action). On a 
 failure, the reload time takes twice as long.
 
-Suppression: 
-Every player with a ranged weapon gets the option to Suppress as the last 
+===== Suppression ===== 

```

Comment:
I left "Suppression" as a topic because it's in the section of "Ranged Combat". Thataway, "Ranged Combat" is specific to ranged concepts and "Melee Combat" is specific to melee concepts. Maybe "Suppression" can be a sub-topic if that is preferable (do to the length of writing)? The system isn't 100% consistent but we could also move "Suppression" to be under "Other Combat Details", but I think it makes the most sense to put it below "Ranged Combat".

Reply:
done. will be in the next push.